RELEASE

```diff
@@ -20,15 +20,16 @@ The Axom development team is comprised of:
 
 Arlie Capps (capps2@llnl.gov)
 Noah Elliott (elliott22@llnl.gov)
-Josh Essman (essman1@llnl.gov)
+Brian Gunney (gunney1@llnl.gov)
 Brian Han (han12@llnl.gov)
 Cyrus Harrison (harrison37@llnl.gov)
 Rich Hornung (hornung1@llnl.gov)
-Esteban Pauli (pauli2@llnl.gov)
 Lee Taylor (taylor16@llnl.gov)
 Kenny Weiss (kweiss@llnl.gov)
 Chris White (white238@llnl.gov)
 Max Yang (wang39@llnl.gov)
 
 emeritus
 George Zagaris
+Josh Essman
+Esteban Pauli
```

Comment:
Since we're maintaining these lists of names, we should also include Randy Settgast, Aaron Black, and Ben Corbett as emeriti.

Reply:
Also, should we add a list of summer students who have contributed over the years, after all the code they developed eventually made it in? @kennyweiss at al. thoughts?